### ISSUE TITLE
Asking for a feature.

### ISSUE THREAD

**Issue by PedroBenevidesPT** [JaredRombough/Wrestling#229]
Good day sir,

I can't find you on the web anywhere other than on this repo. I was wondering if the project has gone stale.

I've been using your software as a EWR scenario converter and it does wonders. I was wondering if you could implement the conversion of the missing wrestlers attributes, like selling, attitude and such. And if possible also the contract info about push and  alignment.

If you still around of course.

**Comment by JaredRombough** [comment]
We are talking about conversion to an sqlite database file here? It would probably not be too much work to release a standalone converter with the additional attributes, no promises but I will look into it.

**Comment by PedroBenevidesPT** [comment]
Yes, I tried to mess around with the hex converter you built but since Java isn't my main language I wasn't able to compile the files properly, also I always develop on my Linux machine and java with Linux is not a very good combination.

But if you manage to do that, it would be great tool for several mod makers that lost their work over the years.

**Comment by JaredRombough** [comment]
I have created a separate repository for the conversion tool, I will upload a compiled version to the itch page shortly.

https://github.com/JaredRombough/wrestling-data-conversion-tool

**Comment by PedroBenevidesPT** [comment]
Great news man! That tool will be highly appreciated by the community.
